standard/sections/clause_7_normative_text.adoc

```diff
@@ -269,9 +270,9 @@ include::../recommendations/core/REC_extent_temporal.adoc[]
 
 ===== Additional Temporal Extents
 
-A WCMP record is able to communicate additional time instants or periods via the `+properties.extent.temporal+` item property.
-Temporal extents can be fully bound or open in either direction (e.g. until or before.  A `+properties.extent.temporal.resolution+`
-property is able express the temporal granularity at which a given dataset's data is measured/generated/reported.
+A WCMP record can communicate additional time instants or periods via the `+properties.extent.temporal+` item property.
```

Comment:
Remove trailing periods from this bulleted list

Reply:
keeping periods